packages/amplication-client/package.json

```diff
@@ -106,10 +104,7 @@
     "@types/chalk-pipe": "2.0.0",
     "@types/downloadjs": "1.4.3",
     "@types/jest": "24.9.1",
-    "@types/lodash.debounce": "4.0.6",
-    "@types/lodash.groupby": "4.6.6",
-    "@types/lodash.keyby": "4.6.6",
-    "@types/lodash.omit": "4.5.6",
+    "@types/lodash": "^4.14.182",
```

Comment:
Please use exact version because its a service and not a lib, please check the --save-exact npm's flag

Reply:
> Please use exact version because its a service and not a lib, please check the --save-exact npm's flag

updated